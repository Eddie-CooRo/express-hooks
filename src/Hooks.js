import { resolveDispatcher } from './utils';
export function useParam(paramName) {
  const dispatcher = resolveDispatcher();
}
