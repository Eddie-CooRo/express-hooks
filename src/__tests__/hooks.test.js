import { useParam } from '../Hooks';

describe('hooks', () => {
  it('hook throws error when used without dispatcher', () => {
    expect(() => useParam()).toThrowErrorMatchingSnapshot();
  });
});
