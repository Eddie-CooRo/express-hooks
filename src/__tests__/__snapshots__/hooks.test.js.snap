// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hooks hook throws error when used without dispatcher 1`] = `"Express hooks should be used inside of express handlers or middlewares"`;
